### PR TITLE
allow none in f_beta

### LINKED ISSUE #530: FBeta binary average
## 🚀 Feature

Have fbeta binary metric following sklearn

### Motivation

This is used and is the default in sklearn. It can be misleading if people use it without thinking.

### Code example
Screen Shot 2021-09-17 at 6 57 12 PM

```py
from sklearn.metrics import fbeta_score

y_true = torch.tensor([0,1,1,0,1,1])
y_pred_class = torch.tensor([1,0,1,0,1,0])

fbeta_score(y_true, y_pred_class, 1)

from torchmetrics import FBeta
f1 = FBeta(num_classes=2, beta=1)
f1(y_pred_class,y_true)

f1 = FBeta(num_classes=2, beta=1,average='none')
f1(y_pred_class,y_true)
```

### PR DESCRIPTION
**Pull request by Programmer-RD-AI**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to update the docs?
- [x] Did you write any new necessary tests?

## What does this PR do?

Fixes #530

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Tests

`from torchmetrics import FBeta`
`from sklearn.metrics import fbeta_score`
`import torch`
`y_true = torch.tensor([0, 1, 1, 0, 1, 1])`
`y_pred_class = torch.tensor([1, 0, 1, 0, 1, 0])`
`print(fbeta_score(y_true, y_pred_class, 1))`
`f1 = FBeta(num_classes=2, beta=1)`
`print(f1(y_pred_class, y_true))`
`f1 = FBeta(num_classes=2, beta=1, average='micro')`
`print(f1(y_pred_class, y_true))`

`0.5714285714285715`
`tensor(0.5714)`
`tensor(0.5000)`

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
> how can I add this to the package test ?

in this case, you can a) add doctest, as we have just in the docstring
https://github.com/PyTorchLightning/metrics/blob/430dafae990bb3b725ace6f1ec3bf22844e36aa9/torchmetrics/functional/classification/f_beta.py#L54
or add a completely new parameter in pytest testing with expected values
https://github.com/PyTorchLightning/metrics/blob/master/tests/functional/test_classification.py

**Comment by Borda** [comment]
@Programmer-RD-AI how is it going here? any progress with adding tests? :rabbit:

**Comment by Borda** [comment]
@Programmer-RD-AI any progress here? :rabbit:

**Comment by Programmer-RD-AI** [comment]
hi @Borda,
I am sorry but I don't have any updates.
I have had a lot of other work.
Again I am sorry.
I am going to try and finish this PR.
but if I cant I will close this PR.
Again I am sorry.
With best regards,
Ranuga
